INSTALL.md

```diff
@@ -268,8 +268,92 @@ Building on Windows
 
 **Method 1 - from source**
 
-I really need someone to write correct, modern docs about how to build
-from source on Windows.
+You will need to have Git, CMake and Visual Studio installed.
+
+The minimal set of dependencies for OIIO is: Boost, zlib, libTIFF, OpenEXR, and libjpeg or libjpeg-turbo. If you have them built somewhere then you skip
+the section below, and will only have to point OIIO build process so their locations.
+
+* Boost: get the boost source archive, extract into `{BOOST_ROOT}`.
+  ```
+  cd {BOOST_ROOT}
+  bootstrap
+  b2
+  ```
+* zlib: this will build it, and then delete the non-static library, so they don't get picked up:
+  ```
+  cd {ZLIB_ROOT}
+  git clone https://github.com/madler/zlib .
+  mkdir build
+  cd build
+  cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=. ..
+  cmake --build . --config Release --target install
+  del lib\zlib.lib
+  ```
+* libTIFF:
+  ```
+  cd {TIFF_ROOT}
+  git clone https://gitlab.com/libtiff/libtiff.git .
+  cd build
+  cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=. ..
+  cmake --build . --target install
+  ```
+* libjpeg-turbo:
+  ```
+  cd {JPEG_ROOT}
+  git clone https://github.com/libjpeg-turbo/libjpeg-turbo .
+  mkdir build
+  cd build
+  cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_SHARED=OFF -DCMAKE_INSTALL_PREFIX=. ..
+  cmake --build . --config Release --target install
+  ```
+* OpenEXR: you'll have to point it to your `{ZLIB_ROOT}` location from the above. If copy-pasting the multi-line command (with lines ending in `^`) into
+  cmd.exe prompt, make sure to copy all the lines at once.
+  ```
+  cd {EXR_ROOT}
+  git clone https://github.com/AcademySoftwareFoundation/openexr .
+  cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=dist ^
+    -DBUILD_TESTING=OFF -DBUILD_SHARED_LIBS=OFF -DOPENEXR_BUILD_TOOLS=OFF ^
+    -DOPENEXR_INSTALL_TOOLS=OFF -DOPENEXR_INSTALL_EXAMPLES=OFF ^
+    -DZLIB_ROOT={ZLIB_ROOT}\build ..
+  cmake --build . --target install --config Release
+  ```
+
+Now get the OIIO source and do one-time CMake configuration step. Replace `{*_ROOT}` below with folders where you have put the 3rd party
+dependencies.
+```
+cd {OIIO_ROOT}
```

Comment:
I think it might be more clear to call this OpenImageIO_ROOT, just to be consistent with the CMake convention that somebody would use so their downstream find_package(OpenImageIO) works.